Godeps/_workspace/src/github.com/contiv/contivmodel/servicelb.json

```diff
@@ -43,6 +43,22 @@
                 "items" : "string"
             }
         },
+        "operProperties": {
+          "serviceVip": {
+            "type": "string",
+            "title": "allocated IP addresses"
+          },
+          "numProviders": {
+            "type": "int",
+            "items": "ndpoint",
```

Comment:
I only committed the Godeps here for sanity to pass once. I will be updating godeps and recommiting it.